recipe/meta.yaml

```diff
@@ -14,14 +14,17 @@ source:
 
 
 build:
-  number: 1
+  number: 2
   skip: true        # [win]
   track_features:
     - gf2x
+  run_exports:
+    - {{ pin_subpackage('gf2x') }}
```

Comment:
Looks like gf2x changes soname in major version changes. `pin_subpackage('gf2x', max_pin='x')` is equivalent to `pin_subpackage('gf2x')`